packages/tabs-react/src/Tabs.tsx

```diff
@@ -19,6 +20,8 @@ export const Tabs = (props: Props) => {
     const [tabIDs, setTabIDs] = useState<string[]>([]);
     const [tabPanelIDs, setTabPanelIds] = useState<string[]>([]);
 
+    useEffect(() => props.onChange?.(activeIndex), [activeIndex, props]);
```

Comment:
Her burde kanskje `onChange` være det som er listet opp som en avhengighet, ikke hele `props`? Det er vel en teoretisk mulighet for at f. eks. `className` endrer seg uten at det nødvendigvis betyr at effekten bør kjøre.

Reply:
Eslint klagde av en eller annen grunn når jeg la inn `props.onChange` som en avhengighet. Men i siste commit endret jeg dette siden det jeg først gjorde uansett knakk bygget 😅